cli/cmd/top.go

```diff
@@ -417,7 +426,7 @@ func getTrafficByResourceFromAPI(k8sAPI *k8s.KubernetesAPI, req *pb.TapByResourc
 		close(done)
```

Comment:
The same channel `done` is being closed twice, here and in `pollInput`, hence generating the error
```
panic: close of closed channel

goroutine 29 [running]:
github.com/linkerd/linkerd2/cli/cmd.getTrafficByResourceFromAPI.func1(0xc00004c2a0, 0xc00004c3c0)
        /home/tarun/work/linkerd2/cli/cmd/top.go:426 +0x50
created by github.com/linkerd/linkerd2/cli/cmd.getTrafficByResourceFromAPI
        /home/tarun/work/linkerd2/cli/cmd/top.go:424 +0x2dc
```

Reply:
@Pothulapati Thank you for recommending changes. I have fixed this :)